### PR TITLE
Delete old python

### PR DESCRIPTION
**Pull request by wnichols1**

### CODE REVIEW

**Comment by timcnicholls** [comment]
Requested @ajgdls review then happy to merge

**Comment by ajgdls** [comment]
Looks OK to remove old references.

File: README.md [hunk 1]
```diff
@@ -484,186 +484,4 @@ Available python tools
...
-
-This script will decode datasets of image/reset frames into separate datasets for fine & coarse ADC and
-gain bits. The resulting datasets will be written back into the file. The original raw datasets remain
-un-modified in the file.
-
-The script takes a single argument: the HDF5 file to operate on.
-
-__NOTE__: This script will run on every single dataset found in the root of the file. So only run it once
-and only on a raw datafile from the filewriter.
+0
```
Comment (ajgdls) [inline README.md]: What is this 0 for?  Is this a mistake?

**Comment by ajgdls** [comment]
My goodness I keep messing up, I want to leave a comment instead of approving.  Sorry!

File: README.md (hunk 1 above)
Comment (wnichols1) [reply README.md]: no, it is there to demonstrate that there is meant to be nothing

**Comment by wnichols1** [comment]
> Should you actually just remove the heading as there are now no entries?

well someone may want to add some entries in future. I don't actually understand what is there and so I left it open.

**Comment by ajgdls** [comment]
OK sure, valid reasons.

**Pull request merged**